java-strings/src/main/java/com/baeldung/string/WordCount.java

```diff
@@ -0,0 +1,54 @@
+package com.baeldung.string;
+
+import java.util.StringTokenizer;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ *
+ * @author shwetaJ
+ */
+
+public class WordCount {
+	private static final Logger logger = LoggerFactory.getLogger(WordCount.class);
+
+	static int countUsingCharAt(String stringTocheck, char delimiter) {
```

Comment:
You appear to be using tabs for indentation rather than following the formatting guidelines and using 4 spaces.

Reply:
I'll check it.